addons/website_portal/__manifest__.py

```diff
@@ -14,5 +14,6 @@
     ],
     'data': [
         'views/website_portal_templates.xml',
+        'wizard/portal_wizard_views.xml',
```

Comment:
I put my comment somewhere au pif. Your commit should explain that we keep the wizard but move it in website because portal should not really be used anymore without website. The "why" needs the "how" to win against the "how".